pom.xml

```diff
@@ -410,6 +410,13 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
```

Comment:
i wonder if introducing a new dep is a good idea.

Reply:
It is temporary for now, it makes use of all of itemstacks method so we don't have to reimplement it, once it works we can reimplement everything ourself i guess